File: data/imported/knowledge/slides/net/merged/snippets/snippet_068.cs
[Fact]
    public void TestSlideCountAfterAdd()
    {
        // Adding a slide increases slide count to 2.
        using var pres = new Presentation();
        var layout = pres.LayoutSlides[0];
        pres.Slides.AddEmptySlide(layout);
        pres.Slides.Count.Should().Be(2);
    }

File: data/imported/knowledge/slides/cpp/merged/snippets/snippet_068.cpp
TEST_F(SlidesIntegrationTest, IndexOf) {
    Presentation pres;
    auto* layout = &pres.layout_slides()[0];
    pres.slides().add_empty_slide(layout);
    EXPECT_EQ(pres.slides().index_of(pres.slides()[0]), 0);
    EXPECT_EQ(pres.slides().index_of(pres.slides()[1]), 1);
}